### PR TITLE
[-] PDF : (re) added field note in invoice

### PR DESCRIPTION
**Pull request by mickaelandrieu**

### CODE REVIEW

File: classes/Product.php [hunk 1]
```diff
@@ -889,6 +889,7 @@ public function toggleStatus()
 
     public function delete()
     {
+        $this->deleteSearchIndexes();die;
```
Comment (kpodemski) [inline classes/Product.php]: oops! :-)

Comment (mickaelandrieu) [inline classes/Product.php]: already cleaned, sorry ;)

**Comment by kpodemski** [comment]
:+1: 

Hope that will be merged however it is improvement. I keep fighting to allow us to improvements on 1.6 branch :D

**Comment by mickaelandrieu** [comment]
@kpodemski let's wait for review :)

**Comment by vincentbz** [comment]
Functional validation is ok, code review missing and if ok it will be integrated in the 1.6.1.5 :)

**Comment by julienbourdeau** [comment]
@kpodemski We're rolling back, is it really an improvement ? :D

Also, there is this related PR https://github.com/PrestaShop/PrestaShop/pull/4526

**Comment by kpodemski** [comment]
@julienbourdeau you know what I think, for me such a changes should be approved, and I'm glad to see that this happens :-)

**Comment by mickaelandrieu** [comment]
@julienbourdeau I think we should merge #4526 instead of mine ;)

**Comment by mickaelandrieu** [comment]
ping @lozal2244 I've used your commits, you are now part of this contribution :+1:

**Comment by lozal2244** [comment]
thank you, I appreciate ;)

**Comment by vTerenti** [comment]
QA OK

**Comment by mickaelandrieu** [comment]
:v:

**Comment by Shudrum** [comment]
:+1:

**Comment by catahoula57** [comment]
Hi everyone. Thanks for working on this. My client is using 1.6.1.7 and the customer notes do not show on the invoice. Is this project complete, can I make the mods myself?

**Pull request merged**